resources/views/common/breadcrumbs.html.twig

```diff
@@ -1,19 +1,19 @@
 <nav aria-label="breadcrumb" data-c-breadcrumbs data-c-colour="{{ font_color ?: 'white' }}">
-	<ol>
-		{% for key, url in breadcrumbs %}
+	<ol id="{{ breadcrumbs_id }}">
+		{% for key, url in custom_breadcrumbs ?: breadcrumbs %}
 			{% if loop.first %}
 				<li>
 					<a href="{{ url(url) }}">{{ breadcrumbs_lang.home }}</a>
 					<i class="fas fa-caret-right"></i>
 				</li>
```

Comment:
Why does the first item in the loop need any special behaviour?

Reply:
The breadcrumbs are extracted and built using the URL segments. So the first key in the segments array, is always the locale, homepage (eg. `en => en`). Since, the key is used in the breadcrumbs, this replaces this with the word "Home". I guess I could add it to the list of keys in the lang files if you think that's preferable.

Reply:
Ah, no that makes sense. Maybe just add a comment: "Replace en/fr with a link to Home."